public/home.html

```diff
@@ -1,16 +1,63 @@
-<!-- This is OLD CODE!! -->
 <!DOCTYPE html>
 <html lang="en">
 
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+    <title>Price Chart</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+        integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+    <script src="https://code.jquery.com/jquery.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/chosen/1.5.1/chosen.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/chosen/1.5.1/chosen.jquery.min.js"></script>
 </head>
 
 <body>
-    <h1>TEST</h1>
+    <div class="container">
+        <div class="row">
+            <div class="col-12-md">
+                <h1>Price Chart</h1>
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="col-12-md">
+                <a href="#" class="list-group-item list-group-item-action">AAPL
+                    <div class="d-flex w-100 justify-content-between">
+                        <h3 class="mb-1">$1,010.60</h3>
+                        <p class="mb-1">+$0.15 (0.01%) <small class="text-muted">Today</small></p>
+                        <p class="mb-1">-$0.09 (-0.01%) <small class="text-muted">After Hours</small></p>
+                </a>
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="col-12-md">
+                <canvas id="myChart" width="400" height="400"></canvas>
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="col-12-md">
+                <button type="button" id="OneDayID" onClick="oneDayFn()" class="btn btn-primary"
+                    style="margin-left: 40px">1-Day</button>
+                <button type="button" id="OneMonthID" onClick="oneMonthFn()" class="btn btn-secondary">1-Month</button>
+                <button type="button" id="ThreeMonthID" onClick="threeMonthFn()"
+                    class="btn btn-success">3-Months</button>
+                <button type="button" id="OneYearID" onClick="oneYearFn()" class="btn btn-warning">1-Year</button>
+                <button type="button" id="AllID" onClick="allIdFn()" class="btn btn-danger">ALL</button>
+            </div>
+        </div>
+    </div>
 </body>
 
-</html>
\ No newline at end of file
+</html>
+
+<script src="/js/priceChart-package.js"></script>
```

Comment:
Same here, we don't need to include priceChart-package.js because we installed it to Node